api/base/exceptions.py

```diff
@@ -1,5 +1,35 @@
-from rest_framework.exceptions import APIException
+
 from rest_framework import status
+from rest_framework.exceptions import APIException
+
+
+def json_api_exception_handler(exc, context):
+    """ Custom exception handler that returns errors object as an array """
+
+    from rest_framework.views import exception_handler
+    response = exception_handler(exc, context)
+
+    # Title removed to avoid clash with node "title" errors
+    acceptable_members = ['id', 'links', 'status', 'code', 'detail', 'source', 'meta']
```

Comment:
If I understand what's happening here, this is a list of allowable top-level keys in the `errors` JSON object. If that's the case, I recommend naming this variable `top_level_keys` or similar, to better reflect its purpose. A comment indicating its purpose in more detail would also be helpful.